apps/scan/frontend/src/components/reprint_report_button.tsx

```diff
@@ -17,47 +14,24 @@ interface ReprintReportButtonProps {
 export function ReprintReportButton({
   scannedBallotCount,
   lastPollsTransition,
-  printerInfo,
   isAdditional,
   beforePrint,
   afterPrint,
 }: ReprintReportButtonProps): JSX.Element | null {
-  const machineConfigQuery = getMachineConfig.useQuery();
-  const configQuery = getConfig.useQuery();
-  const scannerResultsByPartyQuery = getScannerResultsByParty.useQuery();
+  const printerStatusQuery = getPrinterStatus.useQuery();
+  const printTallyReportMutation = printTallyReport.useMutation();
 
   async function reprintReport() {
-    assert(machineConfigQuery.data);
-    assert(configQuery.data);
-    const { electionDefinition, precinctSelection, isTestMode } =
-      configQuery.data;
-    assert(electionDefinition);
-    assert(precinctSelection);
-    assert(scannerResultsByPartyQuery.data);
-    assert(lastPollsTransition);
-
     beforePrint();
-    await printReport({
-      pollsTransitionInfo: lastPollsTransition,
-      electionDefinition,
-      precinctSelection,
-      isLiveMode: !isTestMode,
-      machineConfig: machineConfigQuery.data,
-      scannerResultsByParty: scannerResultsByPartyQuery.data,
-      copies: 1,
-      numPagesCallback: afterPrint,
-    });
+    const numPages = await printTallyReportMutation.mutateAsync();
```

Comment:
I need to add error handling here.

Reply:
Actually, looks like it's already wrapped in error handling that serves just fine.